test-infra/ansible/roles/node-identifier/files/identify_and_register.sh

```
@@ -0,0 +1,124 @@
+U_SPEEDS=""
+U_PCIE=""
+
+sanitize() {
+    local val="$1"
+    echo "${val//[^a-zA-Z0-9._-]/}"
+}
+
+append_unique() {
+    local list="$1"
+    local new="$2"
+    if [[ ! " $list " =~ " $new " ]]; then
```

Shellcheck (0.11.0)

[warning] Remove quotes from right-hand side of =~ to match as a regex rather than literally.

(SC2076)